designer/server/src/test/scala/pl/touk/nussknacker/ui/notifications/NotificationServiceTest.scala

```diff
@@ -41,13 +41,13 @@ import scala.util.{Failure, Success, Try}
 class NotificationServiceTest extends AnyFunSuite with Matchers with PatientScalaFutures with MockitoSugar with WithHsqlDbTesting with EitherValuesDetailedMessage with OptionValues with DBIOActionValues {
 
   private implicit val system: ActorSystem = ActorSystem(getClass.getSimpleName)
-  override protected val dbioRunner: DBIOActionRunner = DBIOActionRunner(db)
+  override protected val dbioRunner: DBIOActionRunner = DBIOActionRunner(dbConfig)
```

Comment:
If you already touched `DbConfig` class, WDYT about changing its name? IMO it is misleading now. It suggests that it contains a bean representation of some static, text configuration but in fact, it is a database client with a profile allowing access to specific query constructions.  Maybe `DbSetup` or sth similar?